docs/applications/cloud-storage/access-google-drive-linode.md

```diff
@@ -3,26 +3,26 @@ author:
     name: Scott Sumner
     email: scottinthebooth@gmail.com
 description: 'Access Google Drive from your Linode with Ubuntu 14.04'
-keywords: 'google,drive,console,fuse,apt,ubuntu'
+keywords: ["google", "drive", "console", "fuse", "apt", "ubuntu"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-published: 'Monday, September 28th, 2015'
-modified: 'Thursday, March 17, 2016'
+published: 2015-09-28
+modified: 2016-03-17
 modified_by:
     name: Linode
 title: 'Access Google Drive from Linode with Ubuntu 14.04'
 contributor:
     name: Scott Sumner
 ---
 
-*This is a Linode Community guide. [Write for us](/docs/contribute) and earn $250 per published guide.*
+*This is a Linode Community guide. [Write for us](/content/contribute) and earn $250 per published guide.*
```

Comment:
Check using /docs/contribute will actually work on Hugo. Or maybe just link to the full URL path.

Reply:
@Jskobos Can you look into this?